lib/events/dynamoevents/dynamoevents_test.go

```diff
@@ -62,6 +61,7 @@ func TestMain(m *testing.M) {
 type dynamoContext struct {
 	log   *Log
 	suite test.EventsSuite
+	clock *clockwork.FakeClock
```

Comment:
I need to play with the fake clock in the test, but `suite` contains a `clockwork.Clock` (and some other tests rely on a RealClock), hence the new field.

Reply:
Is type asserting the `clockwork.Clock` into a `*clockwork.FakeClock` not an option for the specific test you've added?